src/scripts.js

```diff
@@ -58,6 +51,27 @@ const fetchAll = () => {
       domUpdates.displayUserHydrationInfo(hydrationRepository, userHydrationData);
       domUpdates.displayUserActivityInfo(activityRepository, userActivities);
     })
+    .catch(error => showGetErrorMsg(error));
+}
+
+const showGetErrorMsg = (error) => {
+  const errorDisplay = document.querySelector('#welcomeUser');
+  errorDisplay.innerText = `Please check your network connection, ${error}`;
+}
+
+const showPostErrorMsg = (formType) => {
+  const activityInputForm = document.querySelector('#activityInputForm');
+  const hydrationInputForm = document.querySelector('#hydrationInputForm');
+  const sleepInputForm = document.querySelector('#sleepInputForm');
+
+  if (formType === "activity") {
+    activityInputForm.innerHTML = `<p>Please check your network connection by refreshing and trying again</p>`;
```

Comment:
If we have time, I may want to go back to this and rephrase, but it's totally fine as is.